extensions/postgres/schema/xdb_sys_schema.sql

```diff
@@ -0,0 +1,61 @@
+CREATE TABLE xdb_sys_current_process_executions(
```

Comment:
what is the purpose of this table?

Reply:
It's for ProcessIdReusePolicy,  checking which executionId is the current/latest one for the same processId. 
https://github.com/xdblab/xdb/wiki/XDB-Proposal-&-Design#xdb_sys_process_current_executions


This is similar to Cadence/Temporal's IdReusePolicy.